packages/theming/atlas/scripts/build.js

```diff
@@ -0,0 +1,61 @@
+const { rm, cd } = require("shelljs");
+const { join } = require("path");
+const concurrently = require("concurrently");
+
+const mode = process.argv[2] || "production";
+const MX_PROJECT_PATH = process.env.ATLAS_MX_PROJECT_PATH; // should be an absolute path.
+const outputDir =
+    mode !== "production" && MX_PROJECT_PATH ? join(MX_PROJECT_PATH, "theme") : join(__dirname, "../dist/theme");
+const projectDeployDir = mode !== "production" && MX_PROJECT_PATH ? join(MX_PROJECT_PATH, "deployment/web") : null;
+
+console.info(`Building for ${mode}...`);
+const watchArg = mode !== "production" ? "--watch" : "";
+const compressArg = mode === "production" ? "--style compressed" : "";
+
+cd(join(__dirname, ".."));
+rm("-rf", outputDir);
+concurrently(
+    [
+        {
+            name: "content-theme",
+            command: `copy-and-watch ${watchArg} 'content/**/*' '${outputDir}'`
+        },
+        {
+            name: "web-sass-and-manifest-theme",
+            command: `copy-and-watch ${watchArg} 'src/web/sass/**/*' '${outputDir}/styles/web/sass'`
+        },
+        {
+            name: "native-manifest-theme",
+            command: `copy-and-watch ${watchArg} src/native/ts/core/manifest.json '${outputDir}/styles/native/core'`
+        },
+        {
+            name: "fonts-theme",
+            command: `copy-and-watch ${watchArg} 'src/web/sass/core/_legacy/bootstrap/fonts/*' '${outputDir}/styles/web/css/fonts'`
+        },
+        {
+            name: "sass-theme",
+            command: `sass ${watchArg} --embed-sources ${compressArg} --no-charset src/web/sass/main.scss '${outputDir}/styles/web/css/main.css'`
+        },
+        {
+            name: "tsc-theme",
+            command: `tsc ${watchArg} --project tsconfig.json --outDir '${outputDir}/styles/native'`
+        }
+    ].concat(
+        projectDeployDir
```

Comment:
I think we can de-duplicate this actions with the one in the list above. Like introduce a function that returns a list

Reply:
@mijay I don't understand, please try explaining differently

Reply:
✅

Reply:
I approached it differently to how you suggested, with a higher order function that helps with differences in _just_ output path.

Reply:
sass compilation is still duplicated. Plus if you introduce such function, you can use it for other cases

Reply:
✅

Reply:
still 2 left 😢